intel_pytorch_extension_py/ops/DNNL_linear_relu.py

```diff
@@ -0,0 +1,55 @@
+import torch
+from torch import nn
+from torch.nn.parameter import Parameter
+from torch.nn import init
+from torch.autograd import Function
+import math
+import _torch_ipex as core
+
+class DNNLFC(Function):

```

Comment:
Are we going to move this into c++?

Reply:
Xiaobing is trying this since the Python way can not work in JIT.